prow/config/templates/config-ConfigMap.yaml

```diff
@@ -83,8 +83,14 @@ data:
             sidecar: {{ .Values.utility_images.sidecar }}
 
     tide:
+      merge_commit_template:
+        "{{ .Values.github.organisation }}":
+          body: >-
+              {{ "{{ .Title }}" }}
+          title: >-
+              {{ "{{ .Body }}" }}
```

Comment:
@RedbackThomson @A-Hilaly LOL. Please see above. :) We have the body and title switched :(

Reply:
Ahh i think was fixed in https://github.com/aws-controllers-k8s/test-infra/pull/28

Reply:
Or maybe it should be something like this?
```
          title: >-
              {{ "{{ .Title }}" }}
          body: >-
              {{ "{{ .Body }}" }}
```

Reply:
> Or maybe it should be something like this?
> 
> ```
>           title: >-
>               {{ "{{ .Title }}" }}
>           body: >-
>               {{ "{{ .Body }}" }}
> ```

yes, this. :)